Source/RakNetSocket2_Berkley.cpp

```diff
@@ -35,8 +35,11 @@ void RNS2_Berkley::SetSocketOptions(size_t recvBufferSize, size_t sendBufferSize
 	r = setsockopt__( rns2Socket, SOL_SOCKET, SO_LINGER, ( char * ) & sock_opt, sizeof ( sock_opt ) );
 	// Do not assert, ignore failure
 
-
-
+#ifdef SO_NOSIGPIPE

```

Comment:
^ It would be possible to make this configurable by the consumer, but it's probably not worth the effort. In which case I would suggest limiting this code block to iOS and keep behaviour in other platforms the same. What do you think?